pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceServiceTest.java

```diff
@@ -721,6 +723,35 @@ public void testModularLoadManagerRemoveInactiveBundleFromLoadData() throws Exce
         });
     }
 
+
+    @Test
+    public void testGetBrokerServiceUrlAsyncWhenBundleUnloading() throws Exception {
```

Comment:
left 2 questions:

1. Why not use 'unload bundle' to trigger the result 'nsdata.get ().isdisabled ()' ?
2. If we need to verify the result "will not close the clientCnx when bundle is being unloaded", I think it is better to use 'clientcnx. close' to verify the result. Or we should change the title to "Lookup Response meta exception when bundle is being unloaded".  Now the test does not match the title.

Reply:
It is hard to add such a unit test. I can change the title to avoid confusion.